drivers/dma/bcm2708-dmaengine.c

```diff
@@ -734,8 +734,6 @@ static int bcm2835_dma_chan_init(struct bcm2835_dmadev *d, int chan_id, int irq)
 	vchan_init(&c->vc, &d->ddev);
 	INIT_LIST_HEAD(&c->node);
 
-	d->ddev.chancnt++;
```

Comment:
It's explained in the commit message.
I'm following up on this commit that came with 3.19: https://github.com/raspberrypi/linux/commit/9eba5536a7434c69d8c185d4bd1c70734d92287d
I guess this is the line that the message refers to: http://lxr.free-electrons.com/source/drivers/dma/dmaengine.c#L875